src/rules.js

```diff
@@ -2,25 +2,27 @@
 
 var fs = require('fs');
 var path = require('path');
-var rules; // Cashing list of rules, so that we only load them once
-
-function getAllRules() {
-  if (!rules) {
-    rules = {};
-    fs.readdirSync(path.join(__dirname, 'rules')).forEach(function(file) {
-      var ruleName = file.replace(/\.js$/, '');
-      rules[ruleName] = require(path.join(__dirname, 'rules', file));
+
+function getAllRules(additionalRulesDirs) {
+  var rules = {};
+  var rulesDirs = [
+    path.join(__dirname, 'rules')
+  ].concat(additionalRulesDirs || []);
+  rulesDirs.forEach(function(rulesDir) {
+    fs.readdirSync(rulesDir).forEach(function(file) {
+      var rule = require(path.join(rulesDir, file));
```

Comment:
I tried this code locally on windows and when I was passing relative paths as additional dirs, this line would break. I think you need to `path.resolve` the `additionalRulesDirs` .
I am using node v6.9.4, I don't know if the behavior of `require` is different in another version.